boards.txt

```diff
@@ -64,6 +65,49 @@ nRF52DK.menu.lfclk.lfrc.build.lfclk_flags=-DUSE_LFRC
 nRF52DK.menu.lfclk.lfsynt=Synthesized
 nRF52DK.menu.lfclk.lfsynt.build.lfclk_flags=-DUSE_LFSYNT
 
+
+nRF52_minidev.name=nRF52 minimalistic devboard
+
+nRF52_minidev.upload.tool=sandeepmistry:openocd
+nRF52_minidev.upload.protocol=
+nRF52_minidev.upload.interface=stlink-v2
```

Comment:
Ok, let remove this line then. Then the user is free to select which programmer to use via the tools -> programmer menu.